libs/java/server_common/src/test/java/com/yahoo/athenz/common/server/notification/NotificationConverterCommonTest.java

```diff
@@ -239,10 +239,10 @@ public void testGetAthenzUIUrl() {
     }
 
     @Test
-    public void testGetWorkflowUrl() {
+    public void testGetDomainWorkflowUrl() {
```

Comment:
We don't have any test cases for the getDomainWorkflowUrl(), so we should have three cases tested:

converter.getDomainWorkflowUrl(null);
converter.getDomainWorkflowUrl("");
converter.getDomainWorkflowUrl("athenz");